tests/models.py

```diff
@@ -9,12 +9,12 @@ class ModelA(models.Model):
 class ModelB(models.Model):
```

Comment:
`flash.tests` module does not required in `flash`package.
So its removed from package.